plasmapy/formulary/lengths.py

```diff
@@ -167,6 +173,9 @@ def gyroradius(
     -----
     One but not both of ``Vperp`` and ``T`` must be inputted.
 
+    If ``lorentzfactor`` is inputted then the previous two can be
+    omitted although at high energies this can lead to rounding errors
+
```

Comment:
In this case, the rounding errors occur when `Vperp` is provided, and it's really close to `c`.